apps/aeutil/lib/scientific.ex

```diff
@@ -1,4 +1,9 @@
 defmodule Aeutil.Scientific do
+  @moduledoc """
+  Converts integer to scientific notation.
+  It's used for the storing the target in the block and recalculating the next targer
```

Comment:
should be "It's used for storing the target in the block and recalculating the next target"